web/src/app/chat/input/ChatInputBar.tsx

```diff
@@ -90,9 +207,45 @@ export function ChatInputBar({
             mx-auto
           "
         >
+          {showSuggestions && filteredPersonas.length > 0 && (
+            <div
+              ref={suggestionsRef}
+              className="text-sm absolute inset-x-0 top-0 w-full transform -translate-y-full "
+            >
+              <div className="rounded-lg py-1.5 bg-white border border-gray-300 overflow-hidden shadow-lg mx-2  px-1.5 mt-2 rounded  z-10">
+                {filteredPersonas.map((currentPersona, index) => (
+                  <button
+                    key={index}
+                    className={`px-2 ${assistantIconIndex == index && "bg-gray-200"} rounded content-start flex gap-x-1 py-1.5 w-full  hover:bg-gray-100 cursor-pointer`}
```

Comment:
nit: I slightly prefer the hover / selected colors to be the same (which is how chatgpt does it)